movie-browser/src/components/Movies.js

```diff
@@ -1,36 +1,56 @@
 import React, { Component } from 'react';
+import Popup from 'reactjs-popup'
+
 
 class Movies extends Component {
-    constructor(props) {
-        super(props);
+        state = {
+            movies: [],
+            shouldShowAllMovies: false
+        };
 
-        this.state = {
-            movies: []
+        componentDidMount() {
```

Comment:
you should use your api's `fetchMovies` here

Reply:
I will refactor into using the API.js component. I wanted the app to work first.